lib/gameteam.rb

```diff
@@ -35,8 +35,28 @@ def self.create_from_csv(game_teams_path)
         shots: row["shots"],
         tackles: row["tackles"]
       }
-    game_teams << GameTeam.new(gameteam_data)
+    @@all << GameTeam.new(gameteam_data)
```

Comment:
I like the decision to switch all the local arrays to class arrays for easier access!